src/ui/layout/Layout.vue

```diff
@@ -266,6 +268,7 @@ export default {
         },
         toggleHasToolbar(selection) {
             let structure = undefined;
+            this.disableClearButton = false;
```

Comment:
This might lead to unintended behavior. Maybe we should instead have the RecentObjectsList component emit an event when adding a single object to an empty list?

Reply:
I'm seeing how to solve this...

Reply:
@ozyx add an event to notify when an object enters the recent list, but I don't know if I put it in the best place...

Reply:
I believe I fixed everything, but let me know if anything went wrong.